lib/api_auth/base.rb

```diff
@@ -22,22 +22,24 @@ class << self
     # access_id: The public unique identifier for the client
     #
     # secret_key: assigned secret key that is known to both parties
-    def sign!(request, access_id, secret_key)
+    def sign!(request, access_id, secret_key, options = {})
+      options = { :override_http_method => nil, :with_http_method => false }.merge(options)
```

Comment:
I was confused, i do want it to be nil by default. The reason I do the merge is just for discoverability.